pkg/virtiofs/virtiofs.go

```diff
@@ -4,10 +4,11 @@ import (
 	"fmt"
 	"path/filepath"
 
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
```

Comment:
nit: any reason to move this above?

Reply:
No, I'm not sure why this happened. Reverted.